packages/components/src/Slider/Slider.scss

```diff
@@ -0,0 +1,49 @@
+.tc-slider-container {
+	:global(label) {
+		display: block;
+	}
+
+	.informations {
+		display: flex;
+		:global(svg) {
+			height: 24px;
+			width: 24px;
+			fill: $brand-primary;
+		}
+
+		.informations-label {
+			margin-left: $padding-small;
+			line-height: 24px;
+		}
+	}
+
+	.icons {
+		display: flex;
+		justify-content: space-between;
+		margin-top: $padding-small;
+		:global(svg) {
+			height: 24px;
+			width: 24px;
+			fill: gray;
+		}
+		.selected {
+			fill: $brand-primary;
+		}
+	}
+}
+
+.tc-slider {
+	:global(.rc-slider-step),
+	:global(.rc-slider-rail),
+	:global(.rc-slider-track) {
+		height: 7px;
```

Comment:
In the guideline, 
```
The slider is following the same specs and behaviors than for switch/toggles
Only its width and color can vary.
```
Can you take the same values as the switch style please (for all sizes and shadows) ?

Reply:
Yes, guidelines were not up to date when i start the devs